wadsrc/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
 cmake_minimum_required( VERSION 2.8.7 )
 
 add_pk3(zdoom.pk3 ${CMAKE_CURRENT_SOURCE_DIR}/static)
+
+set( PK3_INSTALL_PATH share/games/doom CACHE PATH "Directory where zdoom.pk3 will be placed during install." )
+install (FILES "${PROJECT_BINARY_DIR}/zdoom.pk3" DESTINATION share/games/doom)
```

Comment:
You're correct, I'm fixing it now.

Here's what happened: Secounds after I pushed the changes, I decided I wanted to call the variable PK3_INSTALL_PATH, rather than PK3_PATH. In my scramble to fix it, I missed that.